cmd/root.go

```diff
@@ -76,6 +76,8 @@ func Execute() {
 		fmt.Println(err)
 		os.Exit(-1)
 	}
+	// flush after exit
```

Comment:
Nit - I think you mean flush *before* exit in the comment

Reply:
that's right. it is fixed now.